CHANGELOG.md

```diff
@@ -1,9 +1,14 @@
 # Changelog
 
-## [Unreleased]
+## [v0.8.3] - 2024-03-26
 
 - Added a formatted representation of the UpdatePageAction dataclass for more
   human-readable output.
+- If metadata.yaml does not exist, is reads name and doc information from
```

Comment:
Perhaps `If metadata.yaml does not exist, read the name and doc information from the charmcraft.yaml file.`